src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/StartupProjectRegistrar.cs

```diff
@@ -132,7 +133,8 @@ protected override Tasks.Task DisposeCoreAsync(bool initialized)
         {
             foreach (var provider in _launchProviders.Value.Debuggers)
             {
-                if (await provider.Value.CanLaunchAsync(DebugLaunchOptions.DesignTimeExpressionEvaluation)
+                if (string.Equals(DebuggerName, provider.Metadata.DebuggerName, StringComparison.OrdinalIgnoreCase) &&
```

Comment:
Ya, checking for a specific debugger sounds wrong to me. Why do we need to do that? Why can't we check all debuggers? If it's because you are getting a failure trying to get the output path, should this run in a configured project context instead of an unconfigured one?